dirsrvtests/tests/suites/plugins/attruniq_test.py

```diff
@@ -15,8 +18,11 @@
 from lib389.idm.group import Groups
 from lib389._constants import DEFAULT_SUFFIX
 from lib389.topologies import topology_st
+from lib389.utils import ds_is_older
 
-pytestmark = pytest.mark.tier1
+
+pytestmark = [pytest.mark.tier1,
+              pytest.mark.skipif(ds_is_older('1.3.3'), reason="Not implemented")]
```

Comment:
The dirsrvtests/tests/tickets/ticket47823_test.py tests were `tier2`.
Is the change to tier1 intentionally or a typo?